src/dataframe/dataframe.jl

```diff
@@ -793,18 +793,40 @@ julia> deleteat!(df, 2)
    2 │     3      6
 ```
 """
-function Base.deleteat!(df::DataFrame, inds)
-    if !isempty(inds) && size(df, 2) == 0
+Base.deleteat!(df::DataFrame, ::Colon) = empty!(df)
```

Comment:
The docstring describes several methods:
```suggestion
Base.deleteat!(df::DataFrame, inds)

Base.deleteat!(df::DataFrame, ::Colon) = empty!(df)
```